pkg/cloudprovider/providers/openstack/metrics/metrics.go

```diff
@@ -20,87 +20,61 @@ import (
 	"sync"
 	"time"
 
+	openstack_metrics "k8s.io/cloud-provider-openstack/pkg/metrics"
```

Comment:
Please don't use snake_case in golang, for the package names, by convention we are using lower case, single-word names, there should be no need for underscores or mixedCaps. 

So here I'd suggest `cpometrics`, anywhere else that referencing this module could use `openstackmetrics` if the name conflicts.

Reply:
yes, thanks for this ,I will update